packages/apps/app-dashboard/src/components/user-dashboard/consent/AppVersionNotInRegistry.tsx

```diff
@@ -19,7 +19,7 @@ export function AppVersionNotInRegistryConsent({
   appData,
   readAuthInfo,
 }: AppVersionNotInRegistryConsentProps) {
-  const { isDark, toggleTheme } = useSystemTheme();
+  const { isDark, toggleTheme } = useTheme();
```

Comment:
The fact that our theme handling is bleeding into so many different components is a smell.  Theme management is a top-level concern of the entire application, and individual screens and components should have _no awareness of it_.

Let's circle back on this and start using tailwind's theme suppoort for this post-release.